src/model/liveselection.js

```diff
@@ -42,10 +42,41 @@ export default class LiveSelection extends Selection {
 		/**
 		 * Document which owns this selection.
 		 *
-		 * @private
-		 * @member {engine.model.Document} engine.model.Selection#_document
+		 * @protected
+		 * @member {engine.model.Document} engine.model.LiveSelection#_document
 		 */
 		this._document = document;
+
+		/**
+		 * Keeps mapping of attribute name to priority with which the attribute got modified (added/changed/removed)
+		 * last time. Possible values of priority are: `'low'` and `'normal'`.
+		 *
+		 * Priorities are used by internal `LiveSelection` mechanisms. All attributes set using `LiveSelection`
```

Comment:
`low` is being used when attributes are changed indirectly, that is through `_updateAttributes`.

Really, if you think is is overengineered I'll try to close this "logic" (that `_updateAttributes` clears only attributes set by itself) in `_updateAttributes`. IMHO it is over-engineered, but we may resolve this issue in ... spearate issue.